api/server/index.js

```diff
@@ -27,6 +27,13 @@ const startServer = async () => {
   const app = express();
   app.locals.config = paths;
 
+  app.use((req, res, next) => {
+    if (process.env.NO_INDEX === 'true') {
```

Comment:
I would prefer if this is true by default if the user never sets it; also, even though there is not much code, I would also prefer this to be its own standalone middleware module that is imported. Reduces the change to two lines of code in the main server entry (index) file.

Reply:
done